dir.props

```diff
@@ -69,6 +69,8 @@
     <Platform>AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <RunApiCompat>true</RunApiCompat>
+    <!-- Default to using the open public key -->
+    <AssemblyOriginatorKeyFile>$(ToolsDir)Open.snk</AssemblyOriginatorKeyFile>
```

Comment:
I originally set UseOpenKey=true and found that our signing targets always overrode this which is why I switched to using AssemblyOriginatorKeyFile instead. So long as all the assemblies set this property then it should work. I didn't double check all the platform projects, I only spot checked a couple. Were you relying on the default being the MSFT key for any of your projects?